elements/designer-element-palette/designer-element-palette.html

```diff
@@ -96,31 +97,54 @@
 
     _onMouseDown: function(e) {
       let el = e.target;
+      // find the palette item by crawling up until we see a .element
       while (!el.classList.contains('element')) {
         el = el.parentNode;
       }
       let proxy = dragging.createDragProxy(el, true);
       this.$.proxy.innerHTML = proxy.outerHTML;
       let bounds = el.getBoundingClientRect();
+      this._updateDragProxy({
+        left: bounds.left,
+        top: bounds.top,
+        width: el.offsetWidth,
+        height: el.offsetHeight,
+      });
+      this.fire('designer-mini-nav-collapse');
       dragging.startDrag(
         el,
         bounds.left,
         bounds.top,
         e.clientX,
         e.clientY,
-        function(x, y) {
+        function(x, y, e) {
```

Comment:
Also, `e` is added to the set of argument names, but apparently it is never used in this function.